lnbits/db.py

```diff
@@ -65,6 +65,13 @@ def compat_timestamp_placeholder():
         return "?"
 
 
+def get_placeholder(type_: type) -> str:
+    if type_ == datetime.datetime:
+        return compat_timestamp_placeholder()
+    else:
```

Comment:
no need for `else`

Reply:
maybe this?

```py
def get_placeholder(model: Any, field: str) -> str:
   type_ = model.__fields__[field].type_
   ...
```

then call like this:
```py
placeholder = get_placeholder(self.model, self.field)
```

looks cleaner and encapsulates the object reflection logic in one place